pkg/plugin/processor/builtin/impl/avro/schemaregistry/internal/cache.go

```diff
@@ -32,7 +33,7 @@ func newSubjectVersion(subject string, version int) subjectVersion {
 }
 
 func newSubjectFingerprint(subject string, text string) subjectFingerprint {
-	fingerprint := Rabin([]byte(text))
+	fingerprint := schema.Rabin([]byte(text))
```

Comment:
This is a bit of a smell if you ask me, the `Rabin` fingeprint is an implementation detail, that's why it was in the `internal` package before. Seems to me that this cache belongs in the `schema` package. I won't dwell on it though, we can refactor it in a PR down the road when we have a better picture what goes where.

Reply:
I'll add a todo for us here, so we don't forget about it before merging the feature branch to `main`.

Reply:
FTR, Raul will fix this in his PR.

Reply:
Yup https://github.com/ConduitIO/conduit/pull/1630